ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -119,7 +120,8 @@ class IdpLoginScreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      requestState: pot.noneLoading
+      requestState: pot.noneLoading,
+      errorCode: undefined
```

Comment:
you can avoid this, it is undefined by default
Btw you should consider using [Option](https://gcanti.github.io/fp-ts/modules/Option.ts) instead